src/panel-app/apps.component.js

```diff
@@ -24,36 +27,63 @@ export default function Apps(props) {
 
   return (
     <Scoped css={css}>
-      <table className={"table"}>
-        <thead className="table-header">
-          <tr>
-            <th>App Name</th>
-            <th>Status</th>
-            <th>Status Overrides</th>
-          </tr>
-        </thead>
-        <tbody>
+      <span>
```

Comment:
oh interesting. curious: what was the reasoning for the change? just wanted something easier to style or something?

Reply:
Mainly because React was throwing an error saying that `<input/> cannot be a child of <td>` or something... It was late, and I can't remember. It was something about invalid children so I had to use divs and the associated aria roles.